src/interventions/seeding.jl

```diff
@@ -1,37 +1,87 @@
 """
-    distribute_seeded_corals(total_site_area::Vector{Float64},
-        prefseedsites::Vector{Int64}, available_space::Vector{Float64},
+    distribute_seeded_corals(seed_loc_area::Vector{Float64},
+        available_space::Vector{Float64},
         seeded_area::NamedDimsArray)::NamedDimsArray
 
-Calculate proportion of Tabular Acropora (TA) and Corymbose Acropora (CA) to
-be seeded at each of the n_site_int seeding sites selected. Distributes seeded
-corals according to current available space at each selected site.
+Calculate proportion of deployed corals to be seeded at each of the selected locations. 
+Distributes seeded corals according to current available space at each selected site.
 
 # Arguments
-- total_site_area : nsites*1, total area at each site in m²  .
-- prefseedsites : n_site_int*1, indices for the selected seeding sites.
-- available_space : nsites*1, current available space at each site in m².
+- seed_loc_area : area of locations to seed in m².
+- available_space : currently available space at each seed location in m².
 - seeded_area : area (in m²) of each coral type to be seeded with dim taxa.
 
 # Returns
-scaled_seed : NamedDimsArray [taxa to seed ⋅ locations]
+scaled_seed : NamedDimsArray [taxa to seed ⋅ number of seed locations]
 """
-function distribute_seeded_corals(total_site_area::Vector{Float64},
-    prefseedsites::Vector{Int64}, available_space::Vector{Float64},
+function distribute_seeded_corals(seed_loc_area::Vector{Float64},
+    available_space::Vector{Float64},
     seeded_area::NamedDimsArray)::NamedDimsArray
 
-    # Extract site area for sites selected
-    site_area_seed = total_site_area[prefseedsites]
-
     # Proportion of available space on each site relative to total space available on these
     # sites
-    prop_area_avail = available_space[prefseedsites] ./ sum(available_space[prefseedsites])
+    prop_area_avail = available_space ./ sum(available_space)
 
     # Distribute seeded corals (as area) across sites according to available space 
     # proportions:
     #     proportion * (area of 1 coral * num seeded corals)
     # Convert to relative cover proportion by dividing by site area
-    scaled_seed = ((prop_area_avail .* seeded_area') ./ site_area_seed)'
+    scaled_seed = ((prop_area_avail .* seeded_area') ./ seed_loc_area)'
 
     return scaled_seed
 end
+
+"""
+    seed_corals!(cover::Matrix{Float64}, total_location_area::Vector{Float64},
+        leftover_space_m²::Vector{Float64}, seed_locs::BitVector,
+        seeded_area::NamedDimsArray, seed_sc::BitVector, a_adapt::Vector{Float64},
+        Yseed::SubArray, c_dist_t::Matrix{Distribution})::Nothing
+
+Deploy thermally enhanced corals to indicated locations ("seeding" or "outplanting").
+Increases indicated area covered by the given coral taxa and determines the modified
+distribution of critical DHW thresholds.
+
+# Arguments
+- `cover` : Area currently covered by coral
```

Comment:
Maybe add units for these too (cover, total_location_area)

Reply:
Rather than specifying it repeatedly across each argument, I've just added a note in the docstring.